### PR TITLE
feat: application issues

### PR DESCRIPTION
**Pull request by sjaanus**
image

### CODE REVIEW

File: frontend/src/hooks/api/getters/useApplicationOverview/useApplicationOverview.ts [hunk 1]
```diff
@@ -17,7 +23,7 @@ export const useApplicationOverview = (
     );
 
     return {
-        data: data || { environments: [], featureCount: 0, projects: [] },
+        data: data || placeHolderApplication,
```
Comment (sjaanus) [inline frontend/src/hooks/api/getters/useApplicationOverview/useApplicationOverview.ts:26]: Fixing type error

File: frontend/src/component/application/ApplicationOverview.tsx [hunk 2]
```diff
@@ -88,7 +88,7 @@ const EnvironmentHeader = styled(Typography)(({ theme }) => ({
 
 const SuccessStatus = () => (
     <StyledStatus mode='success'>
-        <Check
+        <WarningAmberRounded
```
Comment (sjaanus) [inline frontend/src/component/application/ApplicationOverview.tsx:91]: Use correct icon

File: frontend/src/component/application/ApplicationOverview.tsx [hunk 3]
```diff
@@ -33,41 +33,41 @@ const StyleApplicationContainer = styled(Box)(({ theme }) => ({
...
-        border: '1px solid',
-        borderColor: theme.palette[mode].border,
-        backgroundColor: theme.palette[mode].light,
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
-        padding: theme.spacing(1.5, 3, 2, 3),
-    }),
-);
+const StyledApplicationBox = styled(Box)<{
```
Comment (sjaanus) [inline frontend/src/component/application/ApplicationOverview.tsx:36]: Just linter

File: frontend/src/hooks/api/getters/useApplicationOverview/useApplicationOverview.ts (hunk 1 above)
Comment (kwasniew) [inline frontend/src/hooks/api/getters/useApplicationOverview/useApplicationOverview.ts:26]: placeholderApplication

**Review by kwasniew — APPROVED** [review]
nice

**Pull request merged**